Protos/ExecutionProtocol/data_id.proto

```diff
@@ -0,0 +1,17 @@
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2020 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+
+syntax = "proto3";
+
+/// LLBDataID represents the digest of arbitrary data, and its purpose is to be a handler for interfacing with CAS
+/// systems. LLBDataID does not require the encoding of any particular hash function. Instead, it is expected that the
+/// CAS system itself that provides the digest.
+message LLBDataID {
```

Comment:
Hmm... I think this might need to be named differently.  I expect to add a core `LLBDataID` class.

Reply:
renamed to LLBPBDataID to signal it's the proto version

Reply:
I think thats good, should this be a standard convention? We've already found it is nice to have ability to wrap the pure prototype with a Swift type that makes it more ergonomic (and can sometimes paper over protocol changes)

Reply:
I don't see the value for that now, and just makes the names longer unnecesarily. I added some notes on the llbuild slack here: https://swiftpm.slack.com/archives/CLAD3EFAL/p1590605714079200